packages/grafana-ui/src/components/Spinner/Spinner.story.tsx

```diff
@@ -16,3 +22,19 @@ export const basic = () => {
     </div>
   );
 };
+
+export const withStyle = () => {
+  return (
+    <div>
+      <Spinner
+        className="my-spin-div"
```

Comment:
I think `className` and `iconClassName` would be better moved to the mdx file inside the Preview section to show the usage of available props, since they have no visible effect in this story.